### PR TITLE
feat(PE-3360): added edit ticker config

### PR DESCRIPTION
**Pull request by atticusofsparta**
transaction example: https://viewblock.io/arweave/tx/AZx7E4bJqivL2kFuyKzu7_3bEpsYkSC3qm9kqST3oY0

### CODE REVIEW

**Comment by dtfiedler** [comment]
rebase with develop

File: src/utils/transactionUtils/transactionUtils.ts [hunk 1]
```diff
@@ -185,6 +187,7 @@ export function getArNSMappingByInteractionType(
     transactionData: TransactionData;
   },
 ): ArNSMapping | undefined {
+  let mapping: ArNSMapping = { domain: '' };
```
Comment (dtfiedler) [inline src/utils/transactionUtils/transactionUtils.ts:190]: return within the case

Comment (atticusofsparta) [reply src/utils/transactionUtils/transactionUtils.ts]: added

**Comment by dtfiedler** [comment]
nit: add the ticket number to the github title so JIRA pulls it in

File: src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx [hunk 2]
```diff
@@ -264,6 +264,7 @@ function TransactionWorkflow({
       switch (contractType) {
         case CONTRACT_TYPES.ANT: {
           switch (interactionType) {
+            case ANT_INTERACTION_TYPES.SET_TICKER:
```
Comment (dtfiedler) [inline src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx:267]: interesting we're using a switch here but likely man of these will fall through to do the same thing. let's consider just doing an if check. switches within switches is tough to maintain and comprehend

**Comment by dtfiedler** [comment]
please update the description to include some example TX interactions related to changing a ticker name

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**